src/components/views/Game.js

```diff
@@ -129,7 +130,11 @@ export default function Game() {
                 await new Promise(resolve => setTimeout(resolve, 100));
                 setRounds(responseGame.data.settings.rounds);
                 setRoundsPlayed(responseGame.data.roundsPlayed);
-
+                const timerStarted = localStorage.getItem('timerStarted');
+                if (!timerStarted) {
+                    startTimer();
+                    localStorage.setItem('timerStarted', 'true');
+                }
```

Comment:
I think you can call startTimer() no matter what, with the new backend. It works for me in postman. It would be cool if we could remove the local storage item.